test/python/pulse/test_commands.py

```diff
@@ -50,15 +50,19 @@ def test_can_construct_valid_acquire_command(self):
         self.assertEqual(acq_command.discriminator.params, discriminator_opts)
         self.assertEqual(acq_command.kernel.name, 'boxcar')
         self.assertEqual(acq_command.kernel.params, kernel_opts)
+        self.assertTrue(acq_command.name.startswith('acq'))
```

Comment:
Could you please add a test to make sure the command index is properly incremented.

Reply:
Agree